docs/archive/user/gettingStarted/CreatingAProfile.md

```diff
@@ -8,7 +8,7 @@ of fields and the rules.
 - **Schema Version** - Dictates the method of serialisation of the profile in order for the generator to 
 interpret the profile fields and rules. The latest version is 0.1.
 ```
-    "schemaVersion": "0.1",
```

Comment:
This needs more substantial changes including the deletion of the preceding lines